docs/docs/quick-tour/quick-tour.ipynb

```diff
@@ -815,15 +808,14 @@
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
    "metadata": {
     "id": "zuhVGJNfJVT5"
    },
    "source": [
     "In addition, `fetch()` downloads the resulting DataFrame after performing some checks on how it was obtained and what the data owner authorizes in their policy. These checks allow BastionLab to uphold a decent level of privacy without too big an impact on your workflow.\n",
     "\n",
-    "Recall that the data owner has set the following policy:\n",
+    "Recalling that the data owner has set the following policy:\n",
```

Comment:
Hum, I think Recalling isn't the right word actually. It's too "complicated" for no reason.

"Remember that the data owner has set the following policy:"